components/Common/WithNodeRelease/index.tsx

```diff
@@ -0,0 +1,27 @@
+import { useMemo } from 'react';
+import { useNodeReleases } from '../../../hooks/useNodeReleases';
+import { isNodeRelease } from '../../../util/nodeRelease';
+import type { FC } from 'react';
+import type { NodeRelease, NodeReleaseStatus } from '../../../types';
+
+type WithNodeReleaseProps = {
+  status: NodeReleaseStatus;
+  children: FC<{ release: NodeRelease }>;
+};
+
+export const WithNodeRelease: FC<WithNodeReleaseProps> = ({
+  status,
+  children: Component,
+}) => {
+  const { getReleaseByStatus } = useNodeReleases();
```

Comment:
```suggestion
  const { getReleaseByStatus } = useNodeReleases();

```

Let's add an empty line here